src/jenkins_job_insight/analyzer.py

```
@@ -179,32 +196,231 @@ def _parse_json_response(raw_text: str) -> AnalysisDetail:
-    json_end = text.rfind("}")
-    if json_start != -1 and json_end != -1 and json_end > json_start:
-        json_str = text[json_start : json_end + 1]
+    # Strategy 1: Try parsing the entire text as JSON directly
+    if text.startswith("{"):
         try:
-            data = json.loads(json_str)
+            data = json.loads(text)
             return AnalysisDetail(**data)
-        except Exception:
+        except (json.JSONDecodeError, Exception):
             pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)